lib/core/mergeConfig.js

```diff
@@ -8,9 +8,9 @@ import { forEach, isObject, merge } from 'lodash'
  * @param {Object} config2
  * @returns {Object} New object resulting from merging config2 to config1
  */
-export default function mergeConfig (config1, config2) {
+
+export default function mergeConfig (config1, config2 = {}) {
   // eslint-disable-next-line no-param-reassign
-  config2 = config2 || {}
```

Comment:
`mergeConfig(cfg1, null)` will break it.

Reply:
Thanks @yoyo837 . Yes it will return a null instead empty object. what's your suggestion?